source/clear-linux/guides/telemetrics/telemetry-api.rst

```diff
@@ -0,0 +1,170 @@
+.. _telemetry-api
+
+The telemetry API
```

Comment:
Please revise page title to simply identify the topic e.g. "Telemetry API"

Reply:
Done.